daemon/guestos_mgr.c

```diff
@@ -472,11 +472,6 @@ guestos_mgr_register_newca(unsigned char *cacert, size_t cacertlen)
 	if (!file_is_dir(TRUSTED_CA_STORE))
 		IF_TRUE_RETVAL(dir_mkdir_p(TRUSTED_CA_STORE, 0600), ret);
 
```

Comment:
why do you implicitly revert the latest security bugfixes?

Reply:
PR was created before latest security fixes, rebasing it

Reply:
done